src/components/AuthenticationForm/AuthenticationForm.svelte

```diff
@@ -102,6 +104,55 @@
 
     dispatch('failed')
   }
+
+  function onDemoMode() {
+    saveAuthenticatedUser({
+      email: 'john.doe@email.com',
```

Comment:
`@example.com`, `@example.org` are protected domain dedicated for this kind of usage

Reply:
Fixed 👍